ereolen_support/ereolen_support.helper.inc

```diff
@@ -481,3 +390,65 @@ function _ereolen_support_cleanup_errors() {
     ->condition('timestamp', time() - $month_in_seconds, '<')
     ->execute();
 }
+
+/**
+ * Curl setup for requesting Jira API.
+ *
+ * @param array $requestParts
+ *   An array containing info for the request.
+ *
+ * @return array
+ *   Result wrapped in an array.
+ */
+function _ereolen_support_callApi(array $requestParts): array {
+    $headers = [
+      'Accept: application/json',
+      'Content-Type: application/json',
+    ];
+
+    // Initialize curl.
+    $curl = curl_init();
+
+    // Set method specific options.
+    switch ($requestParts['method']){
+        case 'POST':
+            curl_setopt($curl, CURLOPT_POST, 1);
+
+            if ($requestParts['data'])
+                curl_setopt($curl, CURLOPT_POSTFIELDS, json_encode($requestParts['data']));
```

Comment:
When sending a JSON payload, we usually have to set a `content-type: application/json` header. Doesn't Jira require that?

Reply:
https://github.com/eReolen/ereolen/pull/61/files#diff-2b6da01e67acfbcf2a06cb12807d4058d9feab62d4bdf8a16fb30c615a588511R406